src/systemctl/systemctl.c

```diff
@@ -4488,6 +4488,11 @@ static int print_property(const char *name, sd_bus_message *m, const char *conte
         return 0;
 }
 
+struct property_info {
+                char *load_state;
+                char *active_state;
+};
```

Comment:
This structure definition should be folded into the variable definition below. And `const char *` would be better:
```
struct property_info {
    const char *load_state, *active_state;
} property_info = {};
```